src/panels/ConfigPanel/ConfigPanel.tsx

```diff
@@ -642,6 +646,38 @@ function generateConfigElementList(
           selectedConfig={selectedConfig}
           setSelectedConfig={setSelectedConfig}
           setNewValue={setNewValue}
+          validate={(value) => {
```

Comment:
This is rather nitpicky, but I've observed we are not using the `value` argument, so was wondering if we should remove it or change the name to `_value`?

Reply:
Oops that should have been the config to validate. This should be the same as the config value provided to the function, but I've updated this, just in case there is some kind of timing issue.